### DISCUSSION TITLE
Копирования ячеек через комбинацию клавиш Ctrl+C

### DISCUSSION THREAD

**Discussion by smplseven** [totumonline/totum-ru-issues-and-discussions#373] [Есть идея, а что если...]
Здравствуйте!

Я хотел бы узнать, есть ли планы на добавление функционала копирования ячеек через комбинацию клавиш Ctrl+C при выделении ячейки в таблице. В настоящее время для копирования значения из ячейки необходимо открыть превью и тапнуть на значение или  через дополительные иконки, что занимает дополнительное время - если приходится работать с большим количеством строк.

Данная функциональность облегчила бы работу с данными в таблицах и повысила удобство использования. 

Если такая возможность еще не реализована, я бы хотел узнать, возможно ли ее добавление в будущих версиях Totum . Буду благодарен за любую информацию по этому вопросу.

Спасибо!

**Comment by ttm-alexeypolunin** [comment]
Да — это записано на реализацию где-то там

**Reply by smplseven** [reply]
А можем мы поднять как-то приоритет этой задачу у Вас?

**Reply by ttm-alexeypolunin** [reply]
12 000.00 р. если безнал то акт будет только с факсимиле
